<?xml version="1.0"?>
<launch>
    <!-- SYSTEM MODEL -->
    <!-- vehicle parameters -->
    <param name="m" type="double" value="1.98" />
    <param name="Iz" type="double" value="0.24" />
    <param name="lf" type="double" value="0.125" />
    <param name="lr" type="double" value="0.125" />
    
    <!-- control parameters -->
    <param name="input_gain" type="double" value="0.3" />
    <param name="air_drag_coeff" type="double" value="0.1308" />
    <param name="friction" type="double" value="0.1711" />
    
    <!-- tire parameters -->
    <param name="tire_model/Kf" type="double" value="0.3" />
    <param name="tire_model/Kr" type="double" value="1.25" />
    <param name="tire_model/mu" type="double" value="0.234" />

    <!-- ekf properties -->
    <param name="state_estimation/q_std" type="double" value="0.1" />
    <param name="state_estimation/r_std" type="double" value="0.1" />
    <param name="state_estimation/v_x_min" type="double" value="0.3" />

	<node pkg="ekf_estimation" type="ekf_estimation" name="ekf_estimation" output="screen" />
</launch>
